operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -493,7 +493,7 @@ def validateNonDynamicClusterReferencedByAtLeastOneServer(self, cluster):
     for server in self.env.getDomain().getServers():
       if self.env.getClusterOrNone(server) is cluster:
         return
-    self.addError("The WebLogic configured cluster " + self.name(cluster) + " is not referenced by any servers.")
+    self.addError("The WebLogic configured cluster " + self.name(cluster) + " is not referenced by any servers.  You must have managed servers defined that belong to this cluster")
```

Comment:
Should there be a period after "this cluster" i.e. "this cluster." ?

Reply:
fixed